### ISSUE TITLE
navigator ?

### ISSUE THREAD

**Issue by chandu0101** [mattiamanzati/react-native-web-polyfill#6]
any plans to port react-native navigator for web ?

**Comment by mattiamanzati** [comment]
You mean the NavigatorIOS component? If so yes, it Will be ported, but first Will be ported non ios-specific components!

**Comment by chandu0101** [comment]
nah i mean pure js navigator http://facebook.github.io/react-native/docs/navigator.html#content

> If so yes, it Will be ported

May be we can skip porting NavigatorIOS , http://facebook.github.io/react-native/docs/navigator-comparison.html#content

**Comment by mattiamanzati** [comment]
Yeah, navigator Will be ported, actually atm i have something pretty much similar I call "router", but the api is Simple to Port, its just an array with an event emitter and a component stack.

**Comment by chandu0101** [comment]
> Yeah, navigator Will be ported

cool :+1:

**Comment by chandu0101** [comment]
https://github.com/bh5-js/react-navigator

**Comment by mattiamanzati** [comment]
Glad to see that!
Ill add a Reference to that to the release im planning For this weekend! 👍

**Comment by mattiamanzati** [comment]
Thanks For Your support!
